data/quickstarts/connect-rhosak-notebook-quickstart.yaml

```diff
@@ -8,7 +8,7 @@ spec:
   appName: rhosak
   durationMinutes: 10
   icon: 'images/red-hat.svg'
-  description: A quick step-by-step guide to connecting to Red Hat Streams for Apache Kafka from a Jupyter notebook.
+  description: Connect to Red Hat Streams for Apache Kafka from a Jupyter notebook. 
```

Comment:
I would change to "Connect to Red Hat OpenShift Streams for Apache Kafka from a Jupyter notebook."